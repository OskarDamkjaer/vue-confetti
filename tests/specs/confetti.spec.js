import Confetti from '../../src/confetti';

describe('Confetti', () => {
  it('is initialised with default settings', () => {
    const confetti = new Confetti();
    expect(confetti).toMatchSnapshot();
  });
});
