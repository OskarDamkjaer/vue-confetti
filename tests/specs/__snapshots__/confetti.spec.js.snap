// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Confetti is initialised with default settings 1`] = `
Confetti {
  "H": 0,
  "W": 0,
  "animationId": null,
  "canvas": null,
  "droppedCount": 0,
  "maxParticlesPerFrame": 2,
  "particles": null,
  "particlesPerFrame": 1.5,
  "wind": 0,
  "windChange": 0.01,
  "windPosCoef": 0.002,
  "windSpeed": 1,
  "windSpeedMax": 1,
}
`;
